internal/service/autoscaling/policy.go

```diff
@@ -87,6 +87,276 @@ func ResourcePolicy() *schema.Resource {
 							MaxItems: 1,
```

Comment:
Line 26, please verify that Import works as expected.

Reply:
I also tested with `terraform apply`, and I don't see any errors. I am not sure if that's enough justification to say the import work as expected.